### ISSUE TITLE
Use cryptography version >=35

### ISSUE THREAD

**Issue by billallen256** [nbgallery/pypki3#4]
The `cryptography` package version 35.0.0 changed the behavior of `x509.load_pem_x509_certificate()` which now raises the following when it gets combined key-and-cert PEM data.

```
ValueError: Valid PEM but no BEGIN CERTIFICATE/END CERTIFICATE delimiters. Are you sure this is a certificate?
```

Either find a way to pass only the `BEGIN/END CERTIFICATE` portion, or wait and see if `cryptography` changes the behavior of this function back.

For the time being, `<35` has been added to the project and tox dependencies.

**Comment by billallen256** [comment]
I filed an issue at https://github.com/pyca/cryptography/issues/6351

**Comment by billallen256** [comment]
Resolved in 29d987796657c80ae0e0b8ed870d6887e92f90a0 using the `pem` package.
